node/src/proxy_server/mod.rs

```diff
@@ -2352,7 +2291,7 @@ mod tests {
             let mut subject = ProxyServer::new(
                 main_cryptde,
                 alias_cryptde,
-                true,
+                false,
```

Comment:
I'm a little nervous to see this flag changed. I wonder if it may be cutting out some of the functionality that this test used to exercise. How hard would it be to modify the injected route up above so that it was plausible in a decentralized context? I don't know.